pkg/cmd/openshift/openshift.go

```diff
@@ -109,14 +109,16 @@ func NewCommandOpenShift(name string) *cobra.Command {
 		Run:   cmdutil.DefaultSubCommandRun(out),
 	}
 
+	f := clientcmd.New(pflag.NewFlagSet("", pflag.ContinueOnError))
```

Comment:
When you already have a factory coming from the root command you should use it instead of instantiate a new one, `oc` and `oadm` for example have it.